packages/authentication/README.md

```diff
@@ -1,3 +1,49 @@
 # @loopback/authentication
 
 A LoopBack component for authentication support.
+
+# Overview
+ It demonstrates how to use LoopBack's user models and passport to interact with other authentication providers.
+ User can login using a passport.js strategy, which could include a third party provider.
+
+
+# Installation
+
+Then in your Node.js project root, run:
```

Comment:
I don't think we are using a sentence like this in other places. See https://loopback.io/doc/en/contrib/README-guidelines.html#installation